components/vf-sass-config/mixins/_links.scss

```diff
@@ -25,6 +25,7 @@
 
   &:visited {
     background-color: $vf-link-visited-color;
+    border-color: $vf-link-visited-color;
```

Comment:
For these, should the dev also be able to pass `$vf-link-visited-color` as a param to the mixin?